### PR TITLE
feat: added m3 buttons styles for stock theme

### PR DESCRIPTION
**Pull request by Bozonych**

### CODE REVIEW

**Review by sergeysenja1992 — APPROVED** [review]

**Review by grankovsky — APPROVED** [review]

**Review by andrewkikot — APPROVED** [review]

File: src/styles/_buttons-m3.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+.mat-mdc-unelevated-button {
+    box-shadow: none !important;
+    &:hover {
+        box-shadow: 0 1px 3px 1px rgba(0, 0, 0, 0.15), 0 1px 2px 0 rgba(0, 0, 0, 0.3) !important;
+    }
+}
+.mat-mdc-elevated-button,
+.mat-raised-button .mdc-fab {
+    box-shadow: 0 1px 3px 1px rgba(0, 0, 0, 0.15), 0 1px 2px 0 rgba(0, 0, 0, 0.3) !important;
```
Comment (denisogolubev) [inline src/styles/_buttons-m3.scss:20]: You can use scss variables here:
```scss
$dark-box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.3);
$light-box-shadow: 0 1px 3px 1px rgba(0, 0, 0, 0.15);
$default-box-shadow: $dark-box-shadow, $light-box-shadow !important;

.mat-mdc-unelevated-button {
    box-shadow: none !important;
    &:hover {
        box-shadow: $default-box-shadow;
    }
}

.mat-mdc-elevated-button,
.mat-raised-button .mdc-fab {
    box-shadow: $default-box-shadow;

    &:hover {
        box-shadow: 0 2px 6px 2px rgba(0, 0, 0, 0.15), $dark-box-shadow !important;
    }
}
```

in this way, it's easier to understand that you are using the same styles in different places)

File: src/styles/_buttons-m3.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.mdc-button {
+    border-radius: 100px !important;
```
Comment (denisogolubev) [inline src/styles/_buttons-m3.scss:2]: 100px? This value a bit scared me) Should it be so huge?

**Review by denisogolubev — APPROVED** [review]

File: src/styles/_buttons-m3.scss (hunk 2 above)
Comment (Bozonych) [reply src/styles/_buttons-m3.scss:2]: It`s ok. This value from material design kit) Dont worry)

File: src/styles/_buttons-m3.scss (hunk 1 above)
Comment (Bozonych) [reply src/styles/_buttons-m3.scss]: thank`s, changed)

**Pull request merged**